src/classes/USER_UserService.cls

```diff
@@ -0,0 +1,271 @@
+			// Check if one of the sysAdminUserIds is the Error Notification recipient
+			for(Id currentSysAdminId: sysAdminUserIds) {
+				if(errorSettings.Error_Notifications_To__c == currentSysAdminId) {
+					// Set Error Notification recipient to All Sys Admin
+					errorSettings.Error_Notifications_To__c = ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS;
+					Database.update(errorSettings, true);
+					break;
+				}
+			}
+		}
+	}
+
+
+	/*********************************************************************************************************
+    * @description Determines if a User is the owner of the Scheduled Jobs.
+    * @param sysAdminUserIds The User Ids that were modified.
+    * @return void
+    **********************************************************************************************************/
+	public static void handleScheduledJobsSubmittedByInActiveIsFrozenSysAdmin(Set<Id> sysAdminUserIds) {
+		Boolean breakOuterCircle = false;
+
+		List<CronTrigger> scheduledJobs = retrieveScheduledJobs(sysAdminUserIds);
+
+		for(Id currentSysAdminId : sysAdminUserIds) {
+			for(CronTrigger eachScheduledJob : scheduledJobs) {
+				if(currentSysAdminId == eachScheduledJob.OwnerId) {
+					notifySysAdminsAboutInActiveIsFrozenSysAdminUser();
+					breakOuterCircle = true;
+					break;
+				}
+			}
+			if(breakOuterCircle) {
+				break;	
+			}
+		}
+	}
+
+
+	/*********************************************************************************************************
+    * @description Logs an Error__c record and emails the org's System Administrators.
+    * @return void
+    **********************************************************************************************************/
+	@testVisible
+	private static void notifySysAdminsAboutInActiveIsFrozenSysAdminUser() {
+		logInActiveIsFrozenUserError();
+		sendInActiveIsFrozenUserEmail();
+	}
+
+
+	/*********************************************************************************************************
+    * @description Logs an Error__c record.
+    * @return void
+    **********************************************************************************************************/
+	@testVisible
+	private static void logInActiveIsFrozenUserError() {
+		Error__c inActiveIsFrozenError = new Error__c(Error_Type__c = ERROR_TYPE_SCHEDULED_APEX,
+														Object_Type__c = OBJECT_TYPE_USER,
+														Datetime__c = System.now(), 
+														Full_Message__c = ERROR_MESSAGE_SCHEDULED_APEX);
+
+		Database.insert(inActiveIsFrozenError, true);
+	}
+
+
+	/*********************************************************************************************************
+    * @description Emails the org's System Administrators.
+    * @return void
+    **********************************************************************************************************/
+	@testVisible
+	private static void sendInActiveIsFrozenUserEmail() {
+		ERR_Notifier.sendErrorNotifications(CONTEXT_TYPE_USER);
```

Comment:
you should be passing in ERR_Handler_API.Context.USER, rather than defining your own string for this enum value.

Reply:
Using the ERR_Handler_API.Context.USER. Thanks!